Library/Homebrew/test/dev-cmd/pull_spec.rb

```diff
@@ -22,7 +22,7 @@
       .and output(/Current branch is new\-branch/).to_stderr
       .and be_a_failure
 
-    expect { brew "pull", "--bump", "8" }
+    expect { brew "pull", "--bump", "https://github.com/Homebrew/homebrew-core/pull/8" }
```

Comment:
To be explicit: this is just to fix the test failure on Linux, right? Using `8` will still pull Homebrew/homebrew-core on macOS?

Reply:
Correct.